.tekton/scanner-component-pipeline.yaml

```diff
@@ -269,12 +269,249 @@ spec:
         value: task
       resolver: bundles
 
+  - name: build-container-s390x
```

Comment:
`build-container-s390x`, `ppc64le` and `arm64` should be very similar with the only difference in the `PLATFORM` parameter.

Similarly to GHA, Tekton supports [Matrix](https://tekton.dev/docs/pipelines/matrix/). Could you please try applying that to reduce the amount of copy-paste in these tasks that use `task-buildah-remote-oci-ta`?

Reply:
I did some experiments with matrices at the start, but couldn't get it to work (though I did combine all architectures and had issues with trying to switch around the builder if on amd64) I'll try again